Streetcode/Streetcode.BLL/MediatR/Media/Video/GetByStreetcodeId/GetVideoByStreetcodeIdQueryHandler.cs

```diff
@@ -17,16 +17,15 @@ public GetVideoByStreetcodeIdQueryHandler(IRepositoryWrapper repositoryWrapper,
             _mapper = mapper;
         }
 
-        public async Task<Result<IEnumerable<VideoDTO>>> Handle(GetVideoByStreetcodeIdQuery request, CancellationToken cancellationToken)
+        public async Task<Result<VideoDTO>> Handle(GetVideoByStreetcodeIdQuery request, CancellationToken cancellationToken)
         {
-            var video = await _repositoryWrapper.VideoRepository.GetAllAsync(video => video.StreetcodeId == request.streetcodeId);
-
-            if (video is null)
+            var video = await _repositoryWrapper.VideoRepository.GetSingleOrDefaultAsync(video => video.StreetcodeId == request.streetcodeId);
+            if (video == null)
```

Comment:
- Spacing
   - Insert blank line between db request and null guard;
   - Put `...OrDefault` call on the separate line

- Why use `.SingleOrDefault` instead of `.FirstOrDefault`? Do we really need to ensure that only one associated video exists?

- Always prefer C# 9 `is` syntax instead of error-prone equals one.

Reply:
fixed it